tests/lib/rules/no-shared-component-data.js

```diff
@@ -102,7 +105,18 @@ ruleTester.run('no-shared-component-data', rule, {
           data
         }
       `,
-      parserOptions: { ecmaVersion: 6, sourceType: 'module' }
+      parserOptions
+    },
+    {
+      filename: 'test.vue',
+      code: `
+        export default {
+          data: () => {
+
+          }
+        }
+      `,
```

Comment:
Ah, I was under the impression that `parserOptions` has `parser: "vue-eslint-parser"`. I'm sorry.